tests/flatten.rs

```diff
@@ -165,6 +165,7 @@ fn do_not_suggest_invalid_alts() {
 }
 
 #[test]
+#[cfg(features = "suggestions")]
```

Comment:
```suggestion
#[cfg(feature = "suggestions")]
```

Per clippy, this is invalid as-written.